lib/fizzy/execute.cpp

```diff
@@ -670,27 +669,31 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const uint64_
         case Instr::br_if:
         case Instr::return_:
         {
+            const auto arity = read<uint8_t>(immediates);
```

Comment:
Could also check if making this `uint32_t` makes any difference.

Reply:
I need a commit for it.

Reply:
Pushed variant with `uint32_t` arity.